conf/app.routes

```diff
@@ -179,6 +179,8 @@ GET         /:srn/pension-commencement-lump-sum
 POST        /:srn/pension-commencement-lump-sum                                         controllers.nonsipp.memberreceivedpcls.PensionCommencementLumpSumController.onSubmit(srn: Srn, mode: Mode = NormalMode)
 GET         /:srn/change-pension-commencement-lump-sum                                  controllers.nonsipp.memberreceivedpcls.PensionCommencementLumpSumController.onPageLoad(srn: Srn, mode: Mode = CheckMode)
 POST        /:srn/change-pension-commencement-lump-sum                                  controllers.nonsipp.memberreceivedpcls.PensionCommencementLumpSumController.onSubmit(srn: Srn, mode: Mode = CheckMode)
+GET         /:srn/what-you-will-need-pcls                                               controllers.nonsipp.memberreceivedpcls.WhatYouWillNeedPensionCommencementLumpSumController.onPageLoad(srn: Srn)
```

Comment:
This could go into a new separate route file (e.g. `pcls.routes`), together with the previous three routes (pension-commencement-lump-sum). Please see receivetransfer.Routes.